modules/pam_unix/pam_unix.8.xml

```diff
@@ -371,6 +388,9 @@
             blowfish, gost-yescrypt, and yescrypt password hashing
             algorithms to
             <replaceable>n</replaceable>.
+            Will be ignored for the crypt_default algorithm, which
```

Comment:
The value will be...

Reply:
The actual value may vary depending on the actual default method used by libxcrypt:

* sha512crypt: 5k rounds, real iterations of the underlying algorithm
* bcrypt: 5 rounds, real iterations of the underlying algorithm
* yescrypt: 5 "rounds" (16 MiBytes of RAM in 4096 blocks, 4 KiBytes each): yescrypt doesn't have a single parameter for iterations, but uses several tunable parameters for amount of RAM, CPU cost of computation, etc.  For the sake of interface compatibility with the gesalt family of functions, we "translate" the rounds parameter to a bouquet of internal parameters.  "rounds" < 3 use less than 2 MiBytes of RAM, but need more CPU time and cause more random reads of small (1 KiByte) memory blocks, thus more L{1,2}-cache misses; "rounds" > 3 use between 4 and 256 MiBytes of memory, but need less CPU time and fewer random reads of (4 KiBytes) memory blocks, thus likely more L{1,2} cache hits.  That way yescrypt compensates the time-memory trade-off (TMTO) most other hashing methods are suffering from.  The amount of real iterations is pseudo-randomized by the entropy of the passphrase and used salt.

Reply:
Yes, that's fine. I just wanted to adjust the text: "This option will be ignored when the crypt_default option is used as the default algorithm always uses the value from the system configuration of libcrypt."

Reply:
Fixed.